CONFIG.md

```diff
@@ -0,0 +1,25 @@
+# Pylabrad Configuration
+
+There are a number of environment variables that can be used to configure
+labrad, which pylabrad also checks. These are as follows:
+
+- `LABRADHOST`: The hostname of the machine where the labrad manager is running
+  (default = localhost).
+- `LABRADPORT`: The tcp port on which the labrad manager is listening
+  (default = 7682).
+- `LABRADPASSWORD`: The password to use when connecting to labrad.
+- `LABRAD_TLS`: The transport-layer security (TLS) mode to use. Must be one of
+  the following:
+  - `starttls`: The default; the initial connection is unencrypted and is then
+    upgraded using the STARTTLS mechanism to secure the connection.
+  - `on`: The initial connection is encrypted. In this mode the default port
+    will be `LABRAD_TLS_PORT` instead of `LABRADPORT`.
```

Comment:
This is really meant for a world where the manager is running with real TLS certs signed by a public authority. In that case you can just use TLS from the beginning to secure connections. It is an extra code path, but I don't think keeping this will be too burdensome.